File: integrations/aws-strands/error-code-corpus/typescript/message_boolean_or.ts
/** A message falling back to a literal when the computed one is empty. */
import { EventType, type BaseEvent } from "@ag-ui/core";

export function emit(detail: string): BaseEvent {
  return {
    type: EventType.RUN_ERROR,
    message: detail || "The boolean fallback.",
    code: "MESSAGE_BOOLEAN_OR",
  };
}

File: integrations/aws-strands/error-code-corpus/python/message_boolean_or.py
"""A message falling back to a literal when the computed one is empty."""

from ag_ui.core import EventType, RunErrorEvent


def emit(detail: str) -> RunErrorEvent:
    return RunErrorEvent(
        type=EventType.RUN_ERROR,
        message=detail or "The boolean fallback.",
        code="MESSAGE_BOOLEAN_OR",
    )
